src/ui/pages/Entity.js

```diff
@@ -405,6 +406,24 @@ export default class readEntityEquipment extends Component {
         }
     }
 
+    onChangeClass = newClass => {
+        const entity = this.state[this.settings.entity];
+
+        // TODO: refactor how entityCode is retrieved
+        const entityCode = this.settings.entityCodeProperty === 'number' ? 'EVNT'
```

Comment:
Why not create a map for this? At least a function. Would also make it easier to reuse.

Reply:
Created a map for this.